src/main/webapp/js/instructorFeedbackAjaxRemindModal.js

```diff
@@ -3,6 +3,7 @@ $(document).ready(function() {
         var button = $(event.relatedTarget); // Button that triggered the modal
         var actionlink = button.data('actionlink');
         
+        $('#remindBtn').prop('disabled', true).prop('value', 'Loading...');
```

Comment:
Having such assumption, you are completely right. Requested changes will be applied soon